content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -43,7 +54,7 @@ Search for QuestionnaireResponses that meet supplied query parameters:
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/QuestionnaireResponses?_id=SH-12508041
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/QuestionnaireResponses?patient=SH-12508041
```

Comment:
you can keep this id and if it was by patient, it would be patient=12508041

Reply:
changed back: 6b1e581f4f01eb2d8a1a69a8a0f857da78c0fcc3